packages/itwinui-css/scripts/createComponent.js

```diff
@@ -91,6 +91,10 @@ const demoHtmlFactory = (directory) => {
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>${componentName} | iTwinUI</title>
+    
```

Comment:
Remove this empty line

Reply:
All of the html files have a new line between the `<title>` and import `<script>` tags, so I added the new line for consistency